src/components/dialogs/export-case-dialog.tsx

```diff
@@ -46,14 +48,26 @@ type FormatParameters = {
 };
 
 interface ExportCaseDialogProps {
+    selectedElements: ElementAttributes[];
     onClose: () => void;
-    onExport: (format: string, parameters: FormatParameters) => Promise<void>;
+    onExport: (
+        format: string,
+        parameters: FormatParameters,
+        fileName: string
+    ) => Promise<void>;
 }
 
 const ExportCaseDialog = (props: ExportCaseDialogProps) => {
     const [loading, setLoading] = useState<boolean>(false);
     const [formats, setFormats] = useState<ExportFormats>([]);
     const [selectedFormat, setSelectedFormat] = useState<string | null>(null);
+    const oneFileMode: boolean = useMemo(
+        () => props.selectedElements.length === 1,
```

Comment:
as convention, we now do destructuring at the signature of component, no longer using props.xxx except develop a wrapper component

Reply:
OK